{"filename": "python-keystoneauth1-5.6.0-1-any.pkg.tar.xz", "name": "python-keystoneauth1", "base": "python-keystoneauth1", "version": "5.6.0-1", "desc": "Authentication Library for OpenStack Identity", "csize": "465060", "isize": "4182320", "md5sum": "42789ac4a8e7e97f0826708b3e27b463", "sha256sum": "37c66dcbe9d84455cdf98d629e3dd7ba58eeb42313b61784bace573de9d0e2ea", "pgpsig": "iQIzBAABCAAdFiEEaLNTfzmjE7PldNBndxk/FSvb5qYFAmXlFDUACgkQdxk/FSvb5qaZThAArruT2oSi2uy26DTrptcsmnEYt6n8Sk2QAtEp+4AbTADRtVe1Tr2uq7iWNGhSk1CUrlkWCeYdov0ET8jzxsUK9bXLwUoCetS/f0K7aOPgz50jBoc4+qL014cYAna0E2BOwYjM8lkqwzcNnFhqTUmpJKGkf+ssRZ3weBn12MxI3XLV+PguwvU5cmYDvB8KkKWLuXXzNkEBgWzqIuZDkIwaC2blcIL6ViA3cG6ScmgA42iGF0tQSAQXD2GkeaOGZnfCXn+X5cNomJoM0Xqq5SXanzl/GI8zl5zTWY3I8nh3uStYxp54vOIyCBgL7618pPP4YsXe2ht/aV6WCgyh1t9ApB6Lj7r0G9KrnF0tHNx+cUXDsZC5hVu/Yry0+mKB/NfeQAQ3e6j8m3kX1GgUACy4OmHVAeqdk0rMkJ4kmyCE2Sa8uA+ThcZdTFyifc7ZRHw/DFYu5WDJuukXkks9eFhd7PzzabACg7zHaGMVYmXeDQ7wJNyqfSDe5bRL3LHLJfSdby8/bj5aY8UKApQ/9C6nnyi6oJRc4wLNdVy++XDUx2kumsvEpvrm4a8DpXhq67T3RMWVEzYNpsf3j2NgfVnEOO4xOyCjn6Tt5+TQjnTa8HGW85zGyppJpcV663OJ2c74xhHmFW+cNJJhjqXfMdwK/B1OOYl8P5hLaElx+Bem8XE=", "url": "https://docs.openstack.org/developer/keystoneauth", "license": "Apache", "arch": "any", "builddate": "1709511708", "packager": "Arch Linux ARM Build System <builder+n1@archlinuxarm.org>", "depends": ["python-pbr", "python-iso8601", "python-stevedore", "python-os-service-types", "python-requests"], "optdepends": ["python-lxml: for saml2 support", "python-requests-kerberos: for kerberos support", "python-oauthlib: for oauth1 support", "python-betamax: for betamax support", "python-fixtures: for betamax support"], "makedepends": ["git", "python-setuptools"], "checkdepends": ["python-hacking", "python-oslo-config", "python-oslo-utils", "python-oslotest", "python-betamax", "python-requests-mock", "python-stestr", "python-testresources", "python-testtools", "python-yaml", "python-requests-kerberos", "python-lxml", "python-oauthlib"], "tokens": ["python-keystoneauth1", "python-keystoneauth1_aarch64", "python-keystoneauth1_unstable", "python-keystoneauth1_aarch64_unstable"]}